From e03dbf5c2733c2fa5fe2e207fedf6be124350da5 Mon Sep 17 00:00:00 2001
From: Wang Jianchao <wangjianchao@kuaishou.com>
Date: Mon, 10 Jan 2022 17:10:42 +0800
Subject: [PATCH 09/13] blk: rename ioprio.c to ioprio-common.c

In next patch, blk-ioprio.c is changed to a module named ioprio.ko.
Rename ioprio.c to ioprio-common.c to avoid same ioprio.o in Makefile

Signed-off-by: Wang Jianchao <wangjianchao@kuaishou.com>
---
 block/Makefile                      | 2 +-
 block/{ioprio.c => ioprio-common.c} | 0
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename block/{ioprio.c => ioprio-common.c} (100%)

diff --git a/block/Makefile b/block/Makefile
index 8950913cb..beacc3a03 100644
--- a/block/Makefile
+++ b/block/Makefile
@@ -8,7 +8,7 @@ obj-y		:= bdev.o fops.o bio.o elevator.o blk-core.o blk-sysfs.o \
 			blk-exec.o blk-merge.o blk-timeout.o \
 			blk-lib.o blk-mq.o blk-mq-tag.o blk-stat.o \
 			blk-mq-sysfs.o blk-mq-cpumap.o blk-mq-sched.o ioctl.o \
-			genhd.o ioprio.o badblocks.o partitions/ blk-rq-qos.o \
+			genhd.o ioprio-common.o badblocks.o partitions/ blk-rq-qos.o \
 			disk-events.o blk-ia-ranges.o
 
 obj-$(CONFIG_BOUNCE)		+= bounce.o
diff --git a/block/ioprio.c b/block/ioprio-common.c
similarity index 100%
rename from block/ioprio.c
rename to block/ioprio-common.c
-- 
2.34.1.75.gabe6bb3905

